tests/test_minimal/test_tools/test_expand_paths.py

```diff
@@ -380,3 +381,49 @@ def test_expand_paths_ibl(tmpdir):
 
     tc = unittest.TestCase()
     tc.assertCountEqual(path_expansion_results, expected)
+
+
+class TestConstructPathTemplate(unittest.TestCase):
+    def setUp(self):
+        self.path = "/data/subject456/session123/file.txt"
+
+    def test_normal_operation(self):
+        result = construct_path_template(self.path, subject_id="subject456", session_id="session123")
+        self.assertEqual(result, "/data/{subject_id}/{session_id}/file.txt")
+
+    def test_with_additional_metadata(self):
+        result = construct_path_template(
+            "/data/subject789/session456/image.txt", subject_id="subject789", session_id="session456", file_type="image"
+        )
+        self.assertEqual(result, "/data/{subject_id}/{session_id}/{file_type}.txt")
+
+    def test_empty_subject_id(self):
+        with self.assertRaises(ValueError):
```

Comment:
To ensure the right `ValueError` is raised in the right condition, can you also assert the message expected in each case? (the code itself has 4 separate places where that type is raised, each with different messages)